docs/Cluster-VictoriaMetrics.md

```diff
@@ -545,7 +545,7 @@ Below is the output for `/path/to/vminsert -help`:
   -influxDBLabel string
      Default label for the DB name sent over '?db={db_name}' query parameter (default "db")
   -influxListenAddr string
-     TCP and UDP address to listen for InfluxDB line protocol data. Usually :8189 must be set. Doesn't work if empty. This flag isn't needed when ingesting data over HTTP - just send it to http://<victoriametrics>:8428/write
+     TCP and UDP address to listen for InfluxDB line protocol data. Usually :8089 must be set. Doesn't work if empty. This flag isn't needed when ingesting data over HTTP - just send it to http://<victoriametrics>:8428/write
```

Comment:
Why you think `8089` should be recommended instead `8189`?

Reply:
Seems like the default port that runs the UDP service **usually** is 8089. https://docs.influxdata.com/influxdb/v1.8/administration/ports/

Reply:
Also 8089 used in our docker examples https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/docker-compose.yml#L25